modules/core/include/visp3/core/vpMatrix.h

```diff
@@ -347,8 +347,15 @@ class VISP_EXPORT vpMatrix : public vpArray2D<double>
 #if defined(VISP_HAVE_OPENCV)
   double detByLUOpenCV() const;
 #endif
+  vpMatrix cholesky() const;
+#if defined(VISP_HAVE_EIGEN3)
+  vpMatrix choleskyByEigen3() const;
+#endif
+#if defined(VISP_HAVE_LAPACK)
+  vpMatrix choleskyByLapack() const;
+#endif
 
-  // Compute the exponential matrix of a square matrix
+    // Compute the exponential matrix of a square matrix
```

Comment:
Indentation problem

Reply:
Done in 8dbc865630951c165d3c5cdbecda6d69bfdbda4e